editions/tw5.com/tiddlers/filters/allafter.tid

```diff
@@ -0,0 +1,17 @@
+caption: allafter
+created: 20151017145021839
+creator: matabele
```

Comment:
Many thanks -- noted. Should I now submit a new pull request with creator and modifier fields removed?